tests/pages/LoginsOverviewPage.test.tsx

```diff
@@ -119,7 +125,7 @@ describe("LoginsOverviewPage", () => {
       expect(boxedLink).toHaveLength(3);
     });
 
-    test("it should display primary email primary phone when there are many of each", () => {
+    test("it should display primary email primary phone but all social logins if there are many of each", () => {
       const mockedSortedResponse: SortedIdentities = {
```

Comment:
```suggestion
      const mockedSortedIdentities: SortedIdentities = {
```
You can change this throughout the file

Reply:
OK fixed 2773ef5